blogposts/structural-code-search-at-scale-from-your-browser.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Structural code search. At scale and from your browser.
```

Comment:
```suggestion
title: Going beyond regular expressions with structural code search
```

Mentioning regular expressions sets the context and by providing a comparison, entices them to see how it offers new possibilities.

Reply:
I'm a bit torn about this. I think the 'going beyond regex' is great. On the other hand, there exist tools to do similar things locally, or you can even just run comby locally. Why then bother using Sourcegraph? IMO the problem that Sourcegraph solves is scale (it works large scale, unlike existing tools, and there is some technically interesting parts on this) and convenience (from your browser, on all repos/supported code hosts) and that's important context for structural search.

Reply:
The benefit/answer to "Why bother with Sourcegraph?" might be implicit, and if we're OK with that then I'm OK with the title change.

Reply:
> IMO the problem that Sourcegraph solves is scale

If you also published this blog post externally, such as Medium.com, then your original title makes more sense, but as it's on Sourcegraph's blog, I don't think it's required.

What about leading with a paragraph explaining what Sourcegraph is like our release blog posts., something like:

> Sourcegraph's **Universal Code Search** is used by developers daily at companies such as Uber, Lyft, and Yelp. It works at massive scale, supporting tens of thousands of repositories, all popular languages, with powerful search and filters beyond the capabilities of IDEs, code hosts, and CLI tools.